lib/config.ml

```diff
@@ -67,8 +95,6 @@ module Ciphers = struct
     ]
 
   let supported = default @ [
-    `TLS_DHE_RSA_WITH_AES_256_GCM_SHA384 ;
-    `TLS_DHE_RSA_WITH_AES_128_GCM_SHA256 ;
     `TLS_RSA_WITH_AES_256_GCM_SHA384 ;
     `TLS_RSA_WITH_AES_128_GCM_SHA256 ;
     `TLS_DHE_RSA_WITH_3DES_EDE_CBC_SHA ;
```

Comment:
FWIW here we are announcing RC4 support, which the TLS 1.3 spec explicitly forbids (again, if we do not have a use-case for RC4 it would be sensible to strip support entirely to prevent mistakes):
https://www.rfc-editor.org/rfc/rfc8446.html#appendix-D.5
>    The security of RC4 cipher suites is considered insufficient for the
>   reasons cited in [RFC7465].  Implementations MUST NOT offer or
>   negotiate RC4 cipher suites for any version of TLS for any reason.

Reply:
out of scope for this PR -- yes, we should separately deprecate RC4 (and maybe 3DES?)